Tribler/Core/Modules/restapi/multichain_endpoint.py

```diff
@@ -0,0 +1,77 @@
+import json
+
+from twisted.web import http, resource
+
+from Tribler.community.multichain.community import MultiChainCommunity
+
+
+class MultichainEndpoint(resource.Resource):
+    """
+    This endpoint is responsible for handing requests for multichain data.
+    """
+
+    def __init__(self, session):
+        resource.Resource.__init__(self)
+
+        child_handler_dict = {"stats": MultichainStatsEndpoint}
+
+        for path, child_cls in child_handler_dict.iteritems():
+            self.putChild(path, child_cls(session))
+
+
+class MultichainStatsEndpoint(resource.Resource):
```

Comment:
I agree this should not be categorized as debug. We currently use it as such, but this should eventually be used to give the user an overview of his contribution to the community.